suggestions/browsers.json

```diff
@@ -5,6 +5,34 @@
         "repo": "https://github.com/GrapheneOS/Vanadium",
         "reason": "GrapheneOS only"
     },
+    {
```

Comment:
Firefox cannot be added. We've already added a privacy-friendly alternative called “Mull” (by Divest OS).